recipes/reprounzip-qt/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "reprounzip-qt" %}
+{% set version = "1.0.11" %}
+{% set sha256 = "6738edae930aa8e447ec1c6e1fb0341bbe84ca94ed7e54e14c2abbb3b21af563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-1.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [not py27]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - pyyaml
+    - pyqt <5
```

Comment:
I see that it's not listed in the `setup.py`, but does it really make sense to have `reprounzip-qt` without `reprounzip`?

Reply:
There is only a requirement on the reprounzip command, no import.

Omitting the dependency started as a way to allow installing those from different sources (PyPI reprounzip-qt with reprounzip from installer, etc) but everything has been out for a while and is easily installable from Conda so I don't see why not. I'll add the dependency.